jboss-eap-standalone-rhel8/README.md

```diff
@@ -1,18 +1,18 @@
-# VM-Redhat - JBoss EAP 7 standalone mode
+# VM-Redhat - JBoss EAP 7 on RHEL 8 standalone mode
 <a href="https://portal.azure.com/#create/Microsoft.Template/uri/https%3A%2F%2Fraw.githubusercontent.com%2FSpektraSystems%2Fredhat-mw-cloud-quickstart%2Fmaster%2Fjboss-eap-standalone-rhel8%2Fazuredeploy.json" target="_blank">
     <img src="http://azuredeploy.net/deploybutton.png"/>
 </a>
 <a href="http://armviz.io/#/?load=https%3A%2F%2Fraw.githubusercontent.com%2FMicrosoft%2Fredhat-mw-cloud-quickstart%2Fmaster%2Fjboss-eap-standalone-rhel7%2Fazuredeploy.json" target="_blank">
     <img src="http://armviz.io/visualizebutton.png"/>
 </a>
 
-`Tags: JBoss, EAP, Red Hat,EAP7`
+`Tags: JBoss, EAP, Red Hat, EAP7, RHEL8, Azure, Azure VM, Cluster`
```

Comment:
Please suggest whether to add term 'Cluster' since we are not using Clustering in this template. Currently the tags used for the template are “JBoss, Red Hat, EAP 7.2, RHEL 8.0, Azure, Azure VM, Java EE”.